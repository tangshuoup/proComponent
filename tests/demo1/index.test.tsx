/*
 * @Descripttion:
 * @version:
 * @Author: tangshuo
 * @Date: 2023-02-07 17:14:19
 * @LastEditors: tangshuo
 * @LastEditTime: 2023-02-08 11:51:33
 */
import React from 'react';
import { Demo } from '@tangshuo/pro-components';
import { render } from '@testing-library/react';

describe('Demo', () => {
  it('demo', async () => {
    const wrapper = render(<Demo></Demo>);
    expect(!!wrapper.baseElement.querySelector<HTMLDivElement>('.demo')).toBeTruthy();
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
