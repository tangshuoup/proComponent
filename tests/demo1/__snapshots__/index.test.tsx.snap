// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Demo demo 1`] = `
<DocumentFragment>
  <div
    class="demo"
  >
    2gdg
    <span
      class="child"
    >
      22
    </span>
  </div>
</DocumentFragment>
`;
